code/modules/admin/verbs/varedit.dm

```diff
@@ -351,23 +351,25 @@ GLOBAL_PROTECT(VVpixelmovement)
 
 	switch(alert("Would you like to associate a value with the list entry?",,"Yes","No"))
 		if("Yes")
-			L[var_value] = mod_list_add_ass(O) //hehe
-	if (O)
-		if (O.vv_edit_var(objectvar, L) == FALSE)
-			to_chat(src, "Your edit was rejected by the object.")
-			return
+			L[var_value] = mod_list_add_ass(O)
+
+	if(!O?.vv_edit_var(objectvar, L))
```

Comment:
Were you intending to change behavior here so it tells you the edit was rejected if `O` is null?

Reply:
Not intended, goofed up, fixed